api/src/main/resources/workbench.yaml

```diff
@@ -2036,15 +2036,15 @@ paths:
         - dataSet
       description: >
         Preview data set
-      operationId: "previewQuery"
+      operationId: "generateDataSetPreview"
```

Comment:
since this is done per domain, should the name also specify that like generatePreviewByDomain?

Reply:
I would prefer it specify data set. I like `previewDataSetByDomain` though. Will change to that.